src/reference/asciidoc/amqp.adoc

```diff
@@ -715,9 +716,57 @@ This allows the sender to correlate a confirm (ack or nack) with the sent messag
 
 When the template's `mandatory` property is 'true' returned messages are provided by the callback described in <<amqp-template>>.
 
-Starting with *version 1.4* the `RabbitTemplate` supports the SpEL `mandatoryExpression` property, which is evaluated against each request message, as the root evaluation object, resolving to a `boolean` value.
+Starting with _version 1.4_ the `RabbitTemplate` supports the SpEL `mandatoryExpression` property, which is evaluated against each request message, as the root evaluation object, resolving to a `boolean` value.
 Bean references, such as `"@myBean.isMandatory(#root)"` can be used in the expression.
 
+[[template-batching]]
+===== Batching
+
+Starting with _version 1.4.2_, the `BatchingRabbitTemplate` has been introduced.
+This is a subclass of `RabbitTemplate` with an overridden `send` method that batches messages according to the `BatchingStrategy`; only when a batch is complete is the message sent to rabbitmq.
```

Comment:
s/rabbitmq/RabbitMQ

Not sure that we should stop honoring our vendors :smile: